modules/openapi-generator/src/test/resources/3_0/petstore.yaml

```diff
@@ -18,6 +18,18 @@ tags:
   - name: user
     description: Operations about user
 paths:
+  /foo:
```

Comment:
this pattern causes NPE at https://github.com/OpenAPITools/openapi-generator/pull/1200/files#r224685854